test/Generators/Microsoft.Gen.Logging/Unit/ParserTests.LogProperties.cs

```diff
@@ -387,4 +393,34 @@ partial class LoggerClass
 
         await RunGenerator(Source, DiagDescriptors.LogPropertiesHiddenPropertyDetected);
     }
+
+    [Fact]
+    public async Task DefaultToString()
```

Comment:
We should also check that we don't emit the warning for records and record structs

Reply:
Done.